Packages/Sites/Neos.NeosIo/Resources/Private/Scss/Molecules/_Table.scss

```diff
@@ -218,3 +219,13 @@
 		}
 	}
 }
+
+
+.neos-neosio-event-page table {
+	border-radius: 0;
+
+	td,
+	th {
+		padding: 3px 10px;
+	}
+}
```

Comment:
These styles will be refactored as far as an own schedule nodetype is used and classes can be set.